packages/grafana-ui/src/components/Menu/Menu.story.tsx

```diff
@@ -0,0 +1,104 @@
+import { Story, ComponentMeta } from '@storybook/react';
+import React from 'react';
+
+import { GraphContextMenuHeader } from '..';
+import { StoryExample } from '../../utils/storybook/StoryExample';
+import { VerticalGroup } from '../Layout/Layout';
+
+import { Menu } from './Menu';
+import mdx from './Menu.mdx';
+
+const meta: ComponentMeta<typeof Menu> = {
+  title: 'General/Menu',
+  component: Menu,
+  argTypes: {},
+  parameters: {
+    docs: {
+      page: mdx,
+    },
+    knobs: {
+      disabled: true,
+    },
+    controls: {
+      disabled: true,
+    },
+    actions: {
+      disabled: true,
+    },
+  },
+};
+
+export const Examples: Story = (args) => {
```

Comment:
I know this was already there, but these kinds of stories are not very storybook-friendly, because we can't set things like controls on them for example. Ideally this kind of descriptive examples can be added on the documents tab where we can also explain when they should be used or have each of these `StoryExample`s as a separate story if we want people to be able to play around with them / change props.
This can be done as part of a future iteration on organizing storybook stories though 👍

Reply:
@JoaoSilvaGrafana disagree 100% , after using storybook for years these are the stories that are actually useful ? Both as a developer consuming a library and as developer of the component, being able to see all the major component states and use cases one one screen is amazing , makes making changes to the component and seeing / verifying results super nice , never understood the the controls aspect makes its take ages to test and explore different states. Why waste everyone's time with that????

Reply:
Why would we move it to docs and to separate stories??? The point is to have all the major states in one view to help developers explore what it can do and to see what happens when you make code changes to the component

Reply:
I can understand we have a single stories that uses controls (although fail to understand the use case unless it's a complex component that has infinite combination of states)

Reply:
Maybe it's just different ways we use storybook but I personally don't find stories that have lots of states very useful unless we use something like Chromatic to test the regression of the stories.

I like using the controls (or even without controls a bit more explanation around the story or having a different story per prop) because they tell me exactly which property to change to achieve the variant I need for example. Without the explanation, I have to go and look at the code of the story to find out what was used there.

I think this component is a different one because we are actually showing both `Menu` and `Menu.Item` in the same story, but for instance the only way to find out that the prop used to make the menu item red is called `destructive` without going to the code is to go to the "Story" tab of storybook and looking at the code through there, Storybook should be able to tell us all that without reading any code.

Reply:
This can be something we work on together to find the best balance of both, but during the dev interviews we figured out we are missing a lot of documentation and also explanation of how to use each component, so this can be something we improve on the docs tab and see how that then works with various different types of stories

Reply:
@JoaoSilvaGrafana 

>I like using the controls (or even without controls a bit more explanation around the story or having a different story per prop) because they tell me exactly which property to change to achieve the variant I need for example

Ah how different people are :) I find the controls mostly pointless when developing and testing/tweaking a story. Seeing the component in all its states as I make changes to the code is so nice. having to switch to a bunch of different stories and toggle controls to accomplish the same is a bit cumbersome :)  

> I think this component is a different one because we are actually showing both Menu and Menu.Item in the same story, but for instance the only way to find out that the prop used to make the menu item red is called destructive without going to the code is to go to the "Story" tab of storybook and looking at the code through there, Storybook should be able to tell us all that without reading any code.

So you would have one story for Menu and one for MenuItem? They make no sense in isolation. 
But yes I agree that this Examples story can be improved, it could better explain that the destructive prop is used on the red item. Will update it :)

Reply:
Hi sorry to interupt in this conversation! But I'll throw in my 2c as a Grafana plugin dev. I really appreciate when all the possible props are shown in the storybook so I know at a glance what the component is/isn't capable of. With a lot of the components I end up having to look in the code to see (for example) undocumented props. Or for example can this prop be passed JSX or is it only a string as shown in the storybook.

Reply:
Hey 👋 Thanks for replying to this, we are always interested in how other developers are making use of the tools that Grafana has.
Can I just clarify from your comment, do you normally make use of Controls and props descriptions? I'm asking since we are doing a big push to revamp storybook I am interested to know how you normally use it and what you find useful or not useful! If you have any more feedback feel free to let us know 😄